### PR TITLE
MB-6076 Bump uswds from 2.9.0 to 2.10.0

### PR DESCRIPTION
**Pull request by Shkeating**
## Description

This PR updates the uswds version on the project. Dependabot tried to make this update for us on Dec 21st, but there were some design changes in happo (which I am opening this PR to review) and some issues with the e2e tests (which I am hoping don't happen again, but if they do, I will seek help from an engineer).

slack thread: https://ustcdp3.slack.com/archives/CTNTFJSBA/p1608588824279900

## Reviewer Notes
**Design**:
When you look in Happo, you will see that the icons have changed for the alerts. I don't think this brings about a breaking change. I also want to note, that as of right now, we do not log the icons within the alerts on our *icons* page in storybook, because they get imported with the Alert component and aren't used anywhere else. Because of this, I advise we accept this change and make sure we update the Alert with the correct icon in our Sketch Library.
image

The only other "change" I see in happo does not seem to actually be a change. Personally, I cannot see anything different in the before/after images.
image

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
yarn install
yarn storybook
```

## Code Review Verification Steps

* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-6076?atlOrigin=eyJpIjoiMjE2NmI5NTU0MDliNDJmNGIwNmU3ZmMxOGM0NzQ1MGQiLCJwIjoiaiJ9) for this change
* [Original Dependabot PR](https://github.com/transcom/mymove/pull/5514)
* [slack thread](https://ustcdp3.slack.com/archives/CTNTFJSBA/p1608588824279900)

### CODE REVIEW

**Comment by suzubara** [comment]
This is blocked pending a fix for this issue: https://github.com/uswds/uswds/issues/3909

**Comment by Shkeating** [comment]
Update 1/13: issue still "in progress" on USWDS sprint board

**Review by amanda5iu — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "react-highlight-words": "^0.17.0",
     "react-idle-timer": "^4.5.2",
     "react-imask": "^6.0.5",
+    "react-numeric-input": "^2.2.3",
```
Comment (suzubara) [inline package.json:46]: This package was removed and should not be added back

**Review by suzubara — CHANGES_REQUESTED** [review]
I'm requesting changes on this because we need to wait for USWDS to release a patch version, and will end up updating to (most likely) 2.10.1 in this PR. I want to ensure this PR is not able to be merged until that happens.

File: package.json (hunk 1 above)
Comment (Shkeating) [reply package.json:46]: done! i also labelled do not merge to cover our bases

**Comment by suzubara** [comment]
https://github.com/uswds/uswds/releases/tag/v2.10.1 has been released, so we should be able to try this again @Shkeating

**Comment by suzubara** [comment]
This isn't just the integration tests - starting the application (ie, `make client_run`) results in a JS error right off the bat.

image

I am not sure why, but one possibility is that we have 2 different versions of USWDS installed because ReactUSWDS is not caught up with 2.10 yet. The latest version of ReactUSWDS implements USWDS 2.8.0. @Shkeating IMO we should version lock our USWDS to whatever our version of ReactUSWDS requires, which would mean 2.8.0.

**Comment by reggieriser** [comment]
Just a heads up ... a dependabot update PR came through for uswds 2.10.2 (#6045).  I closed it since we have this PR to work through the 2.10.x upgrade.

**Comment by Shkeating** [comment]
> This isn't just the integration tests - starting the application (ie, `make client_run`) results in a JS error right off the bat.
> 
> image
> 
> I am not sure why, but one possibility is that we have 2 different versions of USWDS installed because ReactUSWDS is not caught up with 2.10 yet. The latest version of ReactUSWDS implements USWDS 2.8.0. @Shkeating IMO we should version lock our USWDS to whatever our version of ReactUSWDS requires, which would mean 2.8.0.

I think that's a good point. Should I just close this PR then? Or should I try to coordinate with the folks currently working on reactuswds to use 2.10 in the next realease (even if they dont have component parity to it yet)?

**Review by YanZ777 — APPROVED** [review]

**Pull request merged**